src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -361,11 +369,9 @@ static void uct_rdmacm_cm_handle_event_connect_response(struct rdma_cm_event *ev
     remote_data.conn_priv_data        = hdr + 1;
     remote_data.conn_priv_data_length = hdr->length;
 
-    status = uct_rdmacm_cm_id_to_dev_addr(event->id, &dev_addr, &addr_length);
+    status = uct_rdmacm_cm_id_to_dev_addr(uct_rdmacm_cm_ep_get_cm(cep),
+                                          event->id, &dev_addr, &addr_length);
     if (status != UCS_OK) {
-        ucs_error("%s client (ep=%p id=%p) failed to process a connect response ",
```

Comment:
maybe keep as diag?

Reply:
`uct_rdmacm_cm_id_to_dev_addr` already prints error message per every type of failure with different log levels, so IMO this one looks redundant

Reply:
yes but from `uct_rdmacm_cm_id_to_dev_addr` you can't tell on which event it failed in order to determine where you are in the connection establishment flow (in case of a failure/debug)

Reply:
restored as diag